srcpkgs/libtorrent-rasterbar/template

```diff
@@ -1,22 +1,32 @@
 # Template file for 'libtorrent-rasterbar'
 pkgname=libtorrent-rasterbar
-version=1.1.13
-revision=3
+version=1.2.2
+revision=1
 build_style=gnu-configure
-configure_args="--enable-examples --enable-python-binding --with-boost=${XBPS_CROSS_BASE}/usr --with-boost-python=boost_python36"
+configure_args="--enable-examples --enable-python-binding
+ --with-boost=${XBPS_CROSS_BASE}/usr
+ --with-boost-python=boost_python${py3_ver//./}"
 hostmakedepends="automake pkg-config intltool libtool python3-devel"
 makedepends="libressl-devel boost-devel geoip-devel python3-devel"
 short_desc="C++ bittorrent library by Rasterbar Software"
 maintainer="Eivind Uggedal <eivind@uggedal.com>"
 license="BSD-3-Clause"
 homepage="https://libtorrent.org/"
 distfiles="https://github.com/arvidn/libtorrent/releases/download/libtorrent-${version//./_}/libtorrent-rasterbar-${version}.tar.gz"
-checksum=30040719858e3c06634764e0c1778738eb42ecd0b45e814afa746329a948ead7
+checksum=e579261d7f0acbe82e9b4ce703cb721627cb8075023f8a26405992f489bc6202
+
+case "$XBPS_TARGET_MACHINE" in
+	armv6l*)
```

Comment:
This should (most probably) include `armv5*`, `mips*`, and `ppc|ppc-musl` as well.

Reply:
Added